### ISSUE TITLE
Quorum Block Explorer Issue on Azure 7Node VM

### ISSUE THREAD

**Issue by mwdcp** [Consensys-Incorporated/quorum#338]
#### System information

Geth version: Geth/v1.7.2-stable-ee498061/linux-amd64/go1.7.3

OS & Version: Ubuntu 16.04.3 LTS (GNU/Linux 4.13.0-1011-azure x86_64)

I've had the 7node system on Azure running and the test transactions work as described in the docs. However, after installing the Quorum Blockchain Explorer (https://github.com/blk-io/blk-explorer-free) I am unable to get the blocks to load in the web site. I am running the website on a remote desktop using the Azure server's public ip and the :5000 port as instructed in the docs (http://x.x.x.x:5000). The website loads, but the status shows as DOWN and no blocks are displayed.

I did have to install the docker-compose utility (apt install docker-compose). And the command does seem to start the environment:

NODE_ENDPOINT=http://localhost:22000 docker-compose -f linux-docker-compose.yaml up -d

Creating network "blkexplorerfree_default" with the default driver
Creating blk-mongodb-free
Creating blk-explorer-ui-free
Creating blk-explorer-free

The website displays this error for the status:
server:
status: DOWN
error: Network Error

I have confirmed that my nodes are running and that I can attached to node1 using both the IPC and RPC addresses.

Thanks

![screen shot 2018-04-09 at 3 07 39 pm](https://user-images.githubusercontent.com/38227683/38520228-cab4cc72-3c07-11e8-8bff-9f52cd6d5e19.png)

**Comment by rjbez17** [comment]
The status: DOWN is a separate issue we ran into. It relates to CORs support. The block explorer has 3 containers, mongo db, block chain backend, and a UI. Every time I've had this issue was because the UI container didn't have networking to the backend container. Hope this helps.

**Comment by mwdcp** [comment]
I made some progress - I can attach to a Quorum network when it's on a remote server, not running locally. So the docker setup is working in that scenario. I also moved the explorer to a windows box running docker for windows. And that's how I was able to explore against my Azure Quorum system. But if I try and connect the Quorum explorer to a locally running Ethereum (private) network, the first step exits with a 1 return code and doesn't connect the the geth node. I've tried referencing localhost, 127.0.01, 0.0.0.0, the server's IP (along with my rpcport) for the NODE_ENDPOINT, but no luck. Also turned off my firewall and tried opening all the ports i see in use.

Here's the startup that fails with exit code 1:

blk-explorer-free |  :: Spring Boot ::        (v1.5.6.RELEASE)
blk-explorer-free |
blk-explorer-free exited with code 1

Thanks for you help so far.
